PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/run/AzureFunctionsHostConfigurationViewModel.kt

```diff
@@ -287,7 +288,11 @@ class AzureFunctionsHostConfigurationViewModel(
                     }
                     projectSelector.project.set(fakeProject)
                     reloadTfmSelector(fakeProject)
-                    resetProperties(exePath, programParameters, workingDirectory)
+                    resetProperties(
+                            exePath = exePath,
+                            programParameters = programParameters,
+                            workingDirectory = workingDirectory,
+                            workerRuntimeSupportsExternalConsole = true)
```

Comment:
Should not we check for isolated worker project here and pass true/false accordingly?

Reply:
This one is a "fallback" e.g. for when project is unloaded. The value doesn't really matter in this case.